plugins/API/API.php

```diff
@@ -449,6 +449,10 @@ public function get($idSite, $period, $date, $segment = false, $columns = false)
      */
     public function getRowEvolution($idSite, $period, $date, $apiModule, $apiAction, $label = false, $segment = false, $column = false, $language = false, $idGoal = false, $legendAppendMetric = true, $labelUseAbsoluteUrl = true, $idDimension = false)
     {
+        // check if site exists
+        $idSite = (int) $idSite;
+        $site = new Site($idSite);
```

Comment:
I'm not sure if this is **really** an issue, but someone could check if a site exists by querying this method.